packages/ui-svelte/package.json

```diff
@@ -4,7 +4,11 @@
   "description": "UI Component Library for Svelte and SvelteKit",
   "type": "module",
   "exports": {
-    "./atoms/*": "./src/components/atoms/*/index.ts",
+    ".": {
```

Comment:
Switched to a root export file as not getting code completion for auto-import and resolving sub components to work correctly.